docker/Dockerfile

```diff
@@ -209,7 +209,6 @@ RUN set -e; \
                 curl \
                 diffutils \
                 dos2unix \
-                dstat \
```

Comment:
It eventually depend on `pcp` in Ubuntu 24.04 and `pcp` requires manual configuration...

Reply:
IIRC there's no pkg that has to be manually configured. Everything can be automated somehow but I won't be surprised the solution is not cleared for early adopters.

Reply:
Reported a bug to Ubuntu https://bugs.launchpad.net/ubuntu/+source/pcp/+bug/2064784

Reply:
revert